Emergence/Contexts/Main Menu/Getting Shows/FeaturedShowEmitter.swift

```diff
@@ -0,0 +1,38 @@
+import UIKit
+
+class FeaturedShowEmitter: NSObject, ShowEmitter {
+    let network: ArtsyProvider<ArtsyAPI>
+    let title: String
+
+    init(title:String, initialShows:[Show], network: ArtsyProvider<ArtsyAPI>) {
+        self.network = network
+        self.title = title
+        self.shows = initialShows
+    }
+
+    var updateBlock: ((shows:[Show]) -> ())?
+    func onUpdate( callback: (shows:[Show]) -> () ) {
```

Comment:
Cool, if you want to clean this up, you can add this to the class:

```swift
typealias UpdateCallback = (shows: [Show])) -> ()
```

And then 

```swift
var updateBlock: UpdateCallback?
func onUpdate(callback: UpdateCallback) { ...
```